package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "sort": "./bin/sort words.json > sorted.json && mv sorted.json words.json",
     "lint": "jsonlint -q words.json",
-    "test": "npm run lint && tape test.js"
+    "test": "npm run lint && tape test.js",
+    "validate": "npm ls"
```

Comment:
I honestly have no clue how this end up in there. Maybe this is related to "tape" or "travis"?
Because I've noticed that https://github.com/rafaelrinaldi/porreta/pull/19 also has this. Any ideas?